test/index.js

```diff
@@ -306,14 +306,47 @@ describe("RedisCache", function () {
     });
   });
 
-  it("should set the value in redis with default ttl if without ttl", function (done) {
+  it("should set the value in redis permanently if maxAge is empty string", function (done) {
+    var target = new RedisCache({ maxAge: ""});
+    target.set("key", "value")
+
+    assert(target.client.cache.key.ttl === null);
+    done()
+  });
+
+  it("should set the value in redis with a ttl if given in options", function (done) {
     var target = new RedisCache({ maxAge: 1000 });
     target.set("key", "value").then(function () {
       target.client.cache.key.ttl.should.equal(1);
       done();
     });
   });
 
+  it("should set the value in redis with a specifically given ttl that overrides global default values", function (done) {
+    var target = new RedisCache({ maxAge: 1000 });
+    target.set("key", "value", 2000).then(function () {
+      target.client.cache.key.ttl.should.equal(2);
+      done();
+    });
+  });
+
+  it("should set the value in redis with a ttl if given as a parsable number in a string", function (done) {
+    var target = new RedisCache({ maxAge: "1000"});
+    target.set("key", "value").then(function () {
+      target.client.cache.key.ttl.should.equal(1);
+      done();
+    });
+  });
+
+  it("should throw on initialization if maxAge is a unparsable strings", function (done) {
+    try {
+      new RedisCache({ maxAge: "unknown"});
```

Comment:
Vad skulle hända om jag skickade in `{ maxAge: "-Infinity" }` eller `{ maxAge: "Infinity" }` till cachen för 
```js
isNaN("-Infinity")
false
isNaN("Infinity")
false
```

Reply:
ah shiiiit

Reply:
ska vi bara regex matcha och se till att alla tecknen är siffror?